tripper/backends/collection.py

```diff
@@ -57,19 +57,43 @@ def __init__(
 
     def triples(self, triple: "Triple") -> "Generator[Triple, None, None]":
         """Returns a generator over matching triples."""
-        for s, p, o in self.collection.get_relations(*triple):
-            yield s, p, parse_object(o)
+        for s, p, o, d in self.collection.get_relations(*triple, rettype="T"):
+            if d:
+                lang = d[1:] if d[0] == "@" else None
+                dt = None if lang else d
+                yield s, p, Literal(o, lang=lang, datatype=dt)
+            else:
+                yield s, p, o
 
     def add_triples(self, triples: "Sequence[Triple]"):
         """Add a sequence of triples."""
         for s, p, o in triples:
+            # Strange complains by mypy - it assumed that
```

Comment:
It is just resently that `mypy` has started to complain about `Literal`. `typing.Literal` is no way a reserved keyword, so I see no reason for the new behaviour of `mypy`. 

At the same time is `Literal` the natural term to use in tripper well established in the RDF community. It makes no sense that tripper should change that, because that an arbitrary tool gives it another meaning. Maybe we could replace `Literal` with `Any` in the annotation of the return value of `parse_object()`.

Reply:
As an alternative to removing the type control; do you think it would solve the problem if we specify the return type of `parse_object()` to use `tripper.Literal` instead of `Literal`?

Reply:
Worth checking

Reply:
Tested it. No, mypy doesn't understand `tripper.Literal`. That is unfortunate, but I don't think that an arbitrary tool like mypy should dictate the naming of classes in tripper. I will change the return type of `parse_object()` to `Any`.